build.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+set -ex
+
+BUILD_VERSION=$(date +%Y%m%d)
+
+if [[ ! -f "OI-hipster-${BUILD_VERSION}-virtualbox.box" ]]; then
```

Comment:
Could we add some option to specify which version we like to build? For example if `VirtualBox` isn't used. Maybe it makes more sense to use an `Makefile` instead of an shell script?